### ISSUE TITLE
Ultimaker extended is extruding terribly

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#279]
The Ultimaker Extended has been struggling with extruding for a week or so.  

Trying a cold-pull to clean the nozzle just resulted in stringy drawn out filament rather than nice clean nozzle-void shapes.

It still seemed to be working for high-quality (i.e. low layer height, slow speed, so able to work with lower extrusion flow) but over the weekend it had clogged again, and has oozed blue plastic into some of the rest of the head.  So I'm marking it as out-of-action until someone can strip down the extruder and clean it out properly.

**Comment by amcewen** [comment]
@Thingomy and I spent a good chunk of #makerday stripping down the head and cleaning things out.  The nozzle was full of lots of gunk, which we got out with a combination of heating the nozzle up and picking the softened gunk out with some wire, rounded off with a bit of soaking it in acetone.

https://ultimaker.com/en/community/9513-underextrusion-need-help-with-this-one#entry83942 was handy for seeing how to dismantle the head.

Now printing much better than it was.
